portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,24 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function fetchVideoData(id) {
+  fetch('/data?videoId=' + id)  // sends a request to /data URl with a cursor and whether we want next page or not
+.then(response => response.json()) // parses the response as JSON
+.then((videoJson) => { // now we can reference the fields in myObject!
+    document.getElementById("video-title").innerText = videoJson.title
```

Comment:
Can we put these constants somewhere hardcoded in Java Script land? Maybe a new FormIDs.js file?

Reply:
I'll look into it!